channel.go

```diff
@@ -212,6 +216,27 @@ func (ch *Channel) RemoveMembers(userIDs []string, message *Message) error {
 	return nil
 }
 
+// ImportChannelMessages is a batch endpoint for inserting multiple messages.
+func (ch *Channel) ImportChannelMessages(messages ...*Message) (*ImportChannelMessagesResponse, error) {
```

Comment:
could you add it to channel interface ?

Reply:
done

Reply:
thanks but we need to rename it to `ImportMessages`, it's already a method of channel

Reply:
good point, fixing it